### PR TITLE
provided additional information for the installation

### PR DESCRIPTION
**Pull request by julienvillegas**
Provided additional information for the installation (some frameworks were missing for example)

### CODE REVIEW

File: google-play-games/ios/README.md [hunk 1]
```diff
@@ -119,5 +131,397 @@ project(":ios") {
...
+		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
+		new LwjglApplication(new MyGdxGame(actionResolverDesktop), config);
+	}
+	
+}
+```
+
+##### ActionResolver Android
+
+No Google Games Services implementation for Android on this tutorial at this point. Calling SignIn when running on Android won't do anythning.
```
Comment (intrigus) [inline google-play-games/ios/README.md:249]: Typo, `anythning`

File: google-play-games/ios/README.md [hunk 2]
```diff
@@ -119,5 +131,397 @@ project(":ios") {
...
+
+public interface ActionResolver {
+
+    public void signIn();
+}
+```
+
+##### ActionResolver Desktop
+
+No Google Games Services implementation for Dektop. Calling SignIn when running on Desktop won't do anythning.
```
Comment (intrigus) [inline google-play-games/ios/README.md:207]: Typo, `anythning`

File: google-play-games/ios/README.md [hunk 3]
```diff
@@ -119,5 +131,397 @@ project(":ios") {
...
+	}
+
+}
+
+```
+
+
+##### ActionResolverHTML
+
+No Google Games Services implementation for HTML on this tutorial. Calling SignIn when running on Android won't do anythning.
```
Comment (intrigus) [inline google-play-games/ios/README.md:294]: Should be "Calling SignIn when running on **HTML** won't do anything" 
Typo, `anythning`

**Review by intrigus — APPROVED** [review]
Looks good besides some typos.

**Comment by florianf** [comment]
Thx, sorry it took so long.

**Pull request merged**